frontend/src/components/Table/Table.tsx

```diff
@@ -96,6 +96,7 @@ export interface TableProps<T extends object = {}> extends TableOptions<T> {
   detailsPanel?: DetailsOptions<T>;
   footer?: boolean;
   hideToolbar?: boolean;
+  manualPagination?: boolean;
```

Comment:
What is manual pagination?

Reply:
Any pagination that doesn't use an api call to request a new page is considered manual pagination.